### PR TITLE
Fix dim>1 implicit bijector

### PR DESCRIPTION
**Pull request by b-remy**
The dimension issue should be fixed now. At least the notebook is running!

Btw I didn't have the rights to edit the already existing pull request here.

### CODE REVIEW

**Review by EiffL — APPROVED** [review]
awesome!

File: sbiexpt/bijectors.py [hunk 1]
```diff
@@ -29,16 +29,16 @@ def newton_solver(f, z_init):
 
 @partial(jax.custom_vjp, nondiff_argnums=(0, 1))
 def fixed_point_layer(solver, f, params, b):
```
Comment (aboucaud) [inline sbiexpt/bijectors.py:31]: Shouldn't the argument `b` be removed from the function call ?

Comment (aboucaud) [reply sbiexpt/bijectors.py:31]: Sorry wrong PR 😄  @EiffL has already done it
